packages/expo-linking/src/Linking.ts

```diff
@@ -264,19 +264,6 @@ export function addEventListener(type: 'url', handler: URLListener): EmitterSubs
   return NativeLinking.addEventListener(type, handler);
 }
 
-/**
- * Remove a handler by passing the `url` event type and the handler.
- * @param type The only valid type is `'url'`.
- * @param handler An [`URLListener`](#urllistener) function that takes an `event` object of the type
- * [`EventType`](#eventype).
- * @see [React Native Docs Linking page](https://reactnative.dev/docs/linking#removeeventlistener).
- *
- * @deprecated Call `remove()` on the return value of `addEventListener()` instead.
- */
-export function removeEventListener(type: 'url', handler: URLListener): void {
```

Comment:
Oh no, I believe we can't do this in already released SDK, it's a breaking change 😞

Reply:
@tsapeta - the problem is that the API that `removeEventListener` calls was already removed in react-native in this release, but we didn't actually remove the wrapper for it here. so developers will get a runtime error rather than type error. https://github.com/expo/expo/issues/20449

Reply:
yeah, that breaking changed was actually supposed to be release with SDK 47

Reply:
Isn't it better to keep our function and console warn/error that this is no-op rather than removing it entirely? Then I wouldn't consider it as a breaking change on our side.

Reply:
Did some investigation on this and it seems that the internal logic that tracked active event listeners was also removed on 0.70, we could potentially fix this ourselves by storing events added through the `addEventListener` method into a global variable but I think that's not really worth it. With that in mind, I ended up adding  a warning to this function letting users know that this was removed and that they should use `remove()`

Reply:
Perfect, thank you for changing that 🙏